codecparsers/bitReader_unittest.cpp

```diff
@@ -48,12 +48,12 @@ BITREADER_TEST(NoOverflow)
     std::vector<uint8_t>& data = largeData();
     const uint32_t nBytes = data.size();
 
-    ASSERT_EQ(0, nBytes % (1024 * 1024 * 512));
+    ASSERT_EQ(0u, nBytes % (1024 * 1024 * 512));
```

Comment:
here is my gcc version `gcc (Ubuntu 4.8.4-2ubuntu1~14.04) 4.8.4`
and I do see` -Wall` and `-Werror` in Makefile.

I will try add some signed and unsigned compare in my h264 decoder
```
CC = gcc
CCDEPMODE = depmode=gcc3
CFLAGS = -g -O2 -Wno-unused-function  -Wno-cpp -Wall -Werror
CPP = gcc -E
CPPFLAGS =
CXX = g++
CXXCPP = g++ -E
CXXDEPMODE = depmode=gcc3
CXXFLAGS = -g -O2 -Wno-unused-function -Wno-cpp -Wall -Werror
CYGPATH_W = echo
DEFS = -DHAVE_CONFIG_H
DEPDIR = .deps
DLLTOOL = false

```